ibis/backends/tests/test_client.py

```diff
@@ -303,23 +303,29 @@ def test_rename_table(con, temp_table, temp_table_orig):
 @mark.notyet(
     ["trino"], reason="trino doesn't support NOT NULL in its in-memory catalog"
 )
-@mark.broken(["snowflake", "flink"], reason="shows not nullable column as nullable")
+@mark.broken(["snowflake"], reason="snowflake shows not nullable column as nullable")
 def test_nullable_input_output(con, temp_table):
     sch = ibis.schema(
         [("foo", "int64"), ("bar", dt.int64(nullable=False)), ("baz", "boolean")]
     )
-    t = con.create_table(
-        temp_table,
-        schema=sch,
-        **{"tbl_properties": {"connector": None}} if con.name == "flink" else {},
-    )
+    t = con.create_table(temp_table, schema=sch)
 
     assert t.schema().types[0].nullable
     assert not t.schema().types[1].nullable
     assert t.schema().types[2].nullable
 
 
-@mark.notimpl(["datafusion", "druid", "flink", "polars"])
+@mark.notimpl(["datafusion", "druid", "polars"])
+@pytest.mark.broken(
+    ["flink"],
+    raises=ValueError,
+    reason=(
+        "table `FUNCTIONAL_ALLTYPES` does not exist"
+        "Note (mehmet): Not raised when only this test function is executed, "
+        "but can be reproduced by running all the test functions in this file."
```

Comment:
This is problematic, because it won't pass CI. We don't currently have an issue when running something in a single file causes that issue.